addons/skin.estuary/xml/Includes_DialogSelect.xml

```diff
@@ -141,6 +141,10 @@
 					<param name="id" value="5" />
 					<param name="label" value="" />
 				</include>
+				<include content="DefaultDialogButton">
+					<param name="id" value="8" />
+					<param name="label" value="" />
+				</include>								
```

Comment:
please remove the trailing whitespace after `</include>`

Reply:
Yeap.